src/monattack.cpp

```diff
@@ -2745,19 +2745,19 @@ bool mattack::nurse_check_up( monster *z )
         if( !z->has_effect( effect_countdown ) ) {
             sounds::sound( z->pos(), 8, sounds::sound_t::speech,
                            string_format(
-                               _( "a soft robotic voice says, \"Come here.  I'll give you a check-up.\"" ) ) );
+                               _( "a soft robotic voice say, \"Come here.  I'll give you a check-up.\"" ) ) );
```

Comment:
`Says` was the right variant.

Reply:
But then it will display : `you hear a soft robotic voice says : "blablabla" ` , it's weird, no ?